soc/ite/ec/common/soc_espi.h

```diff
@@ -14,23 +14,26 @@
 extern "C" {
 #endif
 
-#define ESPI_IT8XXX2_SOC_DEV DEVICE_DT_GET(DT_NODELABEL(espi0))
+#define ESPI_IT8XXX2_SOC_DEV         DEVICE_DT_GET(DT_NODELABEL(espi0))
+#define ESPI_ITE_SOC_DEV             ESPI_IT8XXX2_SOC_DEV
```

Comment:
Nit - do you need both macros?

Reply:
I kept `ESPI_IT8XXX2_SOC_DEV` because it's still used in some parts of espi_it8xxx2.c. So for now, both macros are needed.